addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -282,9 +282,14 @@ export const hotkeyService = {
                 activeElement,
                 bypassEditableProtection: true,
                 callback: () => {
-                    // AAB: not sure it is enough, we might need to trigger all events that occur when you actually click
-                    el.focus();
-                    el.click();
+                    if (document.activeElement) {
+                        document.activeElement.blur();
+                    }
+                    // AAB: not sure it is enough, we might need to trigger all events that occur when you actually
+                    setTimeout(() => {
+                        el.focus();
+                        el.click();
+                    });
```

Comment:
That makes sense. However, as far as I understand, this should work as well (because focus will trigger the blur)

```suggestion
                    el.focus();
                    setTimeout(() => el.click());
```

Reply:
This actually reproduces more closely what really happens when you do a "real" click (modulo the fact that a dozen of other events are also triggered). As we have async handlers on the blur (or change?) event, the subsequent events are triggered on the next stack.

Reply:
I think `el.focus()` is not doing what we expect here, in this case the `el` is a `li` tag which is not focusable, when i tried with `el.querySelector('a').focus();` without `document.activeElement.blur();` it also worked.

What we can do either: 
- we keep what we have now because the purpose of `el.focus();` is to blur the activeElement which we can do explicitly here.
- we `querySelector` any focusable element inside `el` and we trigger `focus` on it.

wdyt ?

Reply:
Ok my bad, you're right.
Then we suggest
```js
if (document.activeElement) {
    document.activeElement.blur();
}
setTimeout(() => el.click());
```
Focusing isn't really necessary here.

Reply:
@aab-odoo done, thanks for your time.